programs/bpf_loader/src/lib.rs

```diff
@@ -1818,7 +1848,7 @@ mod tests {
             Err(InstructionError::NotEnoughAccountKeys),
         );
 
```

Comment:
For all the tests of loader-v2 it would be preferable to just force deactivate the feature instead of altering the tests.

Reply:
yeah. restored.